README.md

```diff
@@ -34,12 +34,19 @@ If the installation of a package was not succesfull there will be information st
 
 
 ## This repo contains two notebooks :
+**These notebooks can be executed in a workflow using [papermill](https://github.com/nteract/papermill).**
```

Comment:
These notebooks can be parametrized using [papermill](https://github.com/nteract/papermill). In this way, we can run the template notebooks automatically locally or in Argo workflow.

Reply:
Updated.